bitnami/discourse/2/debian-11/rootfs/opt/bitnami/scripts/discourse-env.sh

```diff
@@ -92,6 +93,8 @@ export DISCOURSE_CONF_FILE="${DISCOURSE_BASE_DIR}/config/discourse.conf"
 export PATH="${BITNAMI_ROOT_DIR}/common/bin:${BITNAMI_ROOT_DIR}/brotli/bin:${BITNAMI_ROOT_DIR}/git/bin:${PATH}"
 
 # Discourse persistence configuration
+DISCOURSE_SERVE_STATIC_ASSETS="${DISCOURSE_SERVE_STATIC_ASSETS:-true}"
+export DISCOURSE_SERVE_STATIC_ASSETS="${DISCOURSE_SERVE_STATIC_ASSETS:-true}"
 export DISCOURSE_VOLUME_DIR="${BITNAMI_VOLUME_DIR}/discourse"
 export DISCOURSE_DATA_TO_PERSIST="${DISCOURSE_DATA_TO_PERSIST:-plugins public/backups public/uploads}"
 
```

Comment:
Unicorn will fail to start if we use the current default value for DISCOURSE_PORT_NUMBER, please change the value from 3000 to 8080.

```
export DISCOURSE_PORT_NUMBER="${DISCOURSE_PORT_NUMBER:-8080}"
```

In addition, please replace the `ports` value at `docker-compose.yaml`.